Implementacion de la pagina de contacto

cgarriv1510 opened the pull request.

jjannik1 commented on contacto.html:
```diff
@@ -0,0 +1,17 @@
+  <title>Contacto - WebNova</title>
+</head>
+<body>
+  <h1>Algo</h1>
```
Pagina bien hecha, solo el h1 esta mal.

cgarriv1510 committed:
WIP: estructura inicial de contact.html

temp fix: ajustes menores de diseño

fix: formulario funcional para contacto

feat: Implementación de la sección Contacto (Issue #1)
